OpenEdXMobile/src/main/java/org/edx/mobile/view/RegisterActivity.java

```diff
@@ -59,7 +65,9 @@
 import retrofit2.Call;
 
 public class RegisterActivity extends BaseFragmentActivity
-        implements SocialLoginDelegate.MobileLoginCallback {
+        implements SocialLoginDelegate.MobileLoginCallback, RegistrationSelectView.OnSpinnerItemSelectedListener,
+                RegistrationOptionSpinner.OnSpinnerFocusedListener {
```

Comment:
Would be better to move `RegistrationSelectView.OnSpinnerItemSelectedListener` to the next line and do an auto-format afterwards 😉